chart/templates/deployment.yaml

```diff
@@ -30,18 +30,19 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
-        {{- if or .Values.extraArgs .Values.startupConfig}}
+        {{- if or .Values.extraArgs .Values.startupConfig .Values.targetManagerConfig}}
         - args:
         {{- end }}
           {{- if .Values.startupConfig}}
-          - --config
-          - /startupconfig/.sparrow.yaml
-          - --tmconfig 
-          - /startupconfig/tmconfig.yaml
+            - --config
+            - /startupconfig/.sparrow.yaml
+          {{- else if .Values.targetManagerConfig}}
```

Comment:
Please use another if here or add the `tmconfig` parameter to the default values in values.yaml `startupConfig` commented out (and no value)

Reply:
I just saw your comment... I had exactly the same though, that was the problem in the first place. I just added 3 ifs, as all cases are currently valid.

The E2E tests pass now.